federation/executor.go

```diff
@@ -25,8 +26,11 @@ type ExecutorClient interface {
 // subquery on any of the federated servers.
 // The planner allows it to coordinate the subqueries being sent to the federated servers
 type Executor struct {
-	Executors map[string]ExecutorClient
-	planner   *Planner
+	Executors    map[string]ExecutorClient
+	planner      *Planner
+	ticker       *time.Ticker
+	schemaSyncer SchemaSyncer
+	schema       []byte
```

Comment:
what is the purpose of this field?

Reply:
To tell if the schema has changed, but I think we can add it to the schema syncer instead and remove it from here